src/main/java/com/jcabi/github/RtStatus.java

```diff
@@ -42,7 +41,6 @@
  * @version $Id$
  * @since 0.23
  */
-@Immutable
```

Comment:
@carlosmiranda `@Immutable` currently throws an exception when trying to instantiate this class. Presumably it doesn't think `JsonObject` is immutable.